lib/charms/mysql/v0/mysql.py

```diff
@@ -83,7 +83,7 @@ def wait_until_mysql_connection(self) -> None:
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 1
+LIBPATCH = 2
```

Comment:
if [mysql vm pr#20](https://github.com/canonical/mysql-operator/pull/20) lands before this, the lib should be bumped